### PR TITLE
Select Public/Private School [#172422168]

### PR DESCRIPTION
**Pull request by MatthewSchultz**
Children should select whether they are in public or private school.

See [[#172422168](https://www.pivotaltracker.com/story/show/172422168)].

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.string "suid"
+    t.integer "school_type", limit: 2
```
Comment (admande) [inline db/schema.rb:26]: Why does the migration have limit: 1 and the schema limit: 2?

**Comment by admande** [comment]
Looks good, but why is the change to the date tests part of this PR and doesn't seem labelled as an atomic commit? Also, it would be helpful to see screenshots to verify it looks like Figma

File: db/schema.rb (hunk 1 above)
Comment (MatthewSchultz) [reply db/schema.rb:26]: That, honestly, is a good question, given that at no point was it ever 2.

I always set integers used as enums to limit 1 - it (supposedly) causes them to be stored as a one-byte data field, making them super smallish. I'm not sure how or why db:migrate decided to change this on me, let me see if I can dig up a strong answer to this.

**Comment by MatthewSchultz** [comment]
> Looks good, but why is the change to the date tests part of this PR and doesn't seem labelled as an atomic commit?

In a way, yes - I got a bit lazy with my commits. In the end, I was simply going to add my tests for the school_type noise, but then I realized that adding school_type broke all the other assertions, since each one was making valid data separately. Rather than edit each one, I just DRY'ed the test.

**Comment by bencalegari** [comment]
LGTM! FYI @MatthewSchultz I manually rebased and squashed your commits. Not sure how to do this in a clean way in github without force pushing the branch.

**Pull request merged**